packages/bun-uws/src/HttpContext.h

```diff
@@ -524,7 +524,8 @@ struct HttpContext {
 
     /* Listen to port using this HttpContext */
     us_listen_socket_t *listen(const char *host, int port, int options) {
-        auto socket = us_socket_context_listen(SSL, getSocketContext(), host, port, options, sizeof(HttpResponseData<SSL>));
+        int error = 0;
+        auto socket = us_socket_context_listen(SSL, getSocketContext(), host, port, options, sizeof(HttpResponseData<SSL>), &error);
```

Comment:
Where is this error propagated?

Reply:
this would change uWS I think we should but in a followup